ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-security.j2

```diff
@@ -129,6 +129,23 @@
 | {{ policy.name }} | > {{ min_digits }} | > {{ min_length }} | > {{ min_lower }} | > {{ min_special }} | > {{ min_upper }} | < {{ max_repetitive }} | < {{ max_sequential }} |
 {%         endfor %}
 {%     endif %}
+{%     if management_security.shared_secret_profiles is arista.avd.defined %}
+
+### Secret Profiles
+{%         for profile in management_security.shared_secret_profiles | arista.avd.natural_sort("profile") %}
+
+#### {{ profile.profile }}
+
+| Secret Name | Receive Lifetime | Transmit Lifetime | Timezone |
+| ----------- | ---------------- | ----------------- | -------- |
+{%             for secret in profile.secrets | arista.avd.natural_sort("name") %}
+{%                 set local_time = "Local Time" if secret.local_time is arista.avd.defined(true) %}
+{%                 set receive_lifetime = "Infinite" if secret.receive_lifetime.infinite is arista.avd.defined(true) else secret.receive_lifetime.start_date_time ~ " - " ~ secret.receive_lifetime.end_date_time %}
+{%                 set transmit_lifetime = "Infinite" if secret.transmit_lifetime.infinite is arista.avd.defined(true) else secret.transmit_lifetime.start_date_time ~ " - " ~ secret.transmit_lifetime.end_date_time %}
```

Comment:
We try to avoid these inline if/else in jinja, since it becomes much harder to read the template.
Please refactor to reguler if / else block style. Also set the UTC here instead of having different methods.

Reply:
Fixed.